test-packages/ts-template-imports-app/package.json

```diff
@@ -4,7 +4,17 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "test": "glint"
+    "test": "npm-run-all test:*",
+    "test:typecheck": "glint",
```

Comment:
why run both?

Reply:
👍  Will fix.

Reply:
Actually, I did this because the other test apps are running both of these.